src/databricks/sql/client.py

```diff
@@ -733,6 +733,7 @@ def execute(
         self,
         operation: str,
         parameters: Optional[TParameterCollection] = None,
+        async_op=False,
     ) -> "Cursor":
         """
         Execute a query and wait for execution to complete.
```

Comment:
Please also update the description of the method

Reply:
updated the description for async_op